backend/apis/parse-docket-pdf.utils.js

```diff
@@ -84,8 +84,16 @@ function parseCharges(lines, sections) {
   }
 
   walkSection(lines, sections, "CHARGES", addDispositionToCharges);
+  let chargesWithSeverity = charges.map(elm => {
+    let index = elm.description.search(/.[A-Z][a-z]|.\d[a-z]/);
```

Comment:
I spent some time looking at this regex. I don't think it correctly finds the parse index. The suggestions I made in the test file shows examples of how the regex is failing to find the correct index. It's ironic that the one test case we have is one where the all caps vs lower case rule (discussed in https://github.com/JustUtahCoders/utahexpungements.org/issues/95) doesn't actually work 😭 

Here's what I think we should do:
1. Add in another test pdf file that *does* have the all caps vs lowercase distinction for offenseName and severity.
2. Change your code to look for the last uppercase word. Something like this:
```js
const words = charge.description.split(/\s/)
const splitIndex = words.findIndex(word => word.toUpperCase() !== word)
const offenseName = words.slice(0, splitIndex).join(' ')
const severity = words.slice(splitIndex).join(' ')
```
3. Brainstorm what we can do about the weird `SPEEDING 70 in a 65` offense name that has some lowercase letters in it even though offense names usually don't have lowercase letters. This offense name seems to be an exceptional case to me, not the norm. Let's try to find as many other exceptional cases as we can, too, in the test data. But if this is the only one, we might be able to get away with just treating it special.

Reply:
ha yeah it is ironic. I will go through all the pdfs and see if I can find any other exceptional cases. If I find more I will let you know.  
I will work on resolving the issues. Thanks for the feedback @joeldenning !

Reply:
I was going through the pdf's and it looks like if the offenseName starts with SPEEDING followed by a number we get "in a" followed by another number. I would assume (I know its bad to assume) that both numbers before "in a" and after will always be atleast 2 digits long ie SPEEDING 60 in a 30 and that might be a way to deal with these special cases.